recipe/deploy/vendors.php

```diff
@@ -1,15 +1,33 @@
 <?php
 namespace Deployer;
 
+set('vendor_paths', ['{{release_or_current_path}}']);
```

Comment:
But does it hurt? Except adding a setting, that's a feature with no impact that will be completely transparent to all users not interested by this.

To be honest, this one is the main reason of this PR.

Reply:
It definitely make code less readable. Also it’s new config to already big list of configs. 

Tasks in Deployer easily overriden, for a reason.

Reply:
> It definitely make code less readable

Sorry, I can't agree with this.

Any developer can read this, this is pretty self explanatory: 
```php
foreach (get('vendor_paths', []) as $path) {
    run("cd $path && {{bin/composer}} {{composer_action}} {{composer_options}} 2>&1");
}
```

> Also it’s new config to already big list of configs.

Although I'm not happy with it, I can understand much more this one. But the feature we're talking about does not force you to set the added option because it has a default behavior that will satisfy 99% of users.

> Tasks in Deployer easily overriden, for a reason.

And Deployer offers a really nice API for this, thanks! Even if I'm a bit frustated to have to override a whole task/recipe for a tiny missing option.

Reply:
> Sorry, I can't agree with this.

Agree. I mean more stuff to ask question while reading code. Someone may encounter this loop and wondering WTF?

> Although I'm not happy with it, I can understand much more this one. But the feature we're talking about does not force you to set the added option because it has a default behavior that will satisfy 99% of users.

I'm looking on it from the other side: only small percentage of users (~1%) gonna benefit from it. What I'm thinking is not try to cover every possible usage scenario, but create "a framework" for user to implement all they want. 

For example you can simple override only update_code task:
```php
task('deploy:update_code', function () {
    foreach (get('vendor_paths', []) as $path) {
        run("cd $path && {{bin/composer}} {{composer_action}} {{composer_options}} 2>&1");
    }
});
```

Reply:
Thanks for the snippet, that's pretty much what I'm doing already. I thought that a bit more flexibility over this wouldn't hurt but I understand your point of view.

I desperetaly tried to find an issue where someone asked for this and didn't found any 😅 
I'm mostly working with WordPress/Timber and that kind of things are often needed. If anyone stumble upon this comment, please raise your 🙌  to help me grow that 1% use case 😄

I'll remove that `vendor_paths` part soon from the PR.

Reply:
As more requests will come - I'll rereview my view)

Please, create separate PR for other features. Will close this PR for now.